### ISSUE TITLE
Billboarding

### ISSUE THREAD

**Issue by io7m** [io7m/r2#2]
At a minimum, billboarding is a feature of the forward pipeline. It could be added to the deferred pipeline if a valid use case can be found (but it's primarily intended to render thousands of translucent sprites in a single call, so it's probably not so useful on the deferred side).

Billboarded instances require a geometry shader and a per-instance world-space position vector.

**Comment by io7m** [comment]
Depends on #16

**Comment by io7m** [comment]
Don't forget that the positions need to be specified in draw order.
